llvm/test/tools/llvm-debuginfo-analyzer/DWARF/Inputs/ThreadLocalStorage.s

```diff
@@ -0,0 +1,309 @@
+	.text
```

Comment:
Perhaps this test could be minimised further...? It would be nice to reduce this down to just what's needed for the issue.

It may also be worth considering the `yaml2obj` format that some debug info tests use elsewhere.

Reply:
Thanks for your comments.
In the initial patches for `llvm-debuginfo-analyzer` I used `yaml2obj`. But for the latest `WebAssembly` patches, I start using `llvm-mca`.

Reply:
@jryans Changed from using a full `.s` to a `reduced IR`, with just the minimum code to reproduce the issue. Thanks.

Reply:
Thanks, the IR version looks much more minimal and maintainable.